### ISSUE TITLE
Allow Round-Robin Notification Step to Set Importance

### ISSUE THREAD

**Issue by thomasmitchell** [grafana/oncall#1184]
Currently, you can set the type of notification sent (default or important) for other step-types such as Notify Users from Schedule or Notify Users, but there is no such option for the Round Robin notification step.

Additionally, it is unclear what kind of notification will currently be sent if a round-robin notification is triggered.

So this is a feature request for Round-Robin step to allow setting the notification importance level.

https://grafana.zendesk.com/agent/tickets/157288

**Comment by qmonitoring** [comment]
Indeed, there is no mention in the documentation what kind of notification will be sent (Default or Important) if a round-robin escalation step is chosen.

**Comment by ethanmdavidson** [comment]
Just did a quick test and it sent the notification at "default" level, which appears to be, ahem, [the default](https://github.com/grafana/oncall/blob/88f9f118a3f98b5e2b7d37a1a1bd1d6fb2d4d929/engine/apps/alerts/tasks/notify_user.py#L28). (seems obvious in hindsight!)

**Comment by joeyorlando** [comment]
@Ferril is there anything blocking this issue?

**Comment by Konstantinov-Innokentii** [comment]
@iskhakov @mackgorski it seems your opinion is needed here

**Comment by Ferril** [comment]
Related discussions -  https://raintank-corp.slack.com/archives/C06K1MQ07GS/p1729086490888139

**Comment by joeyorlando** [comment]
@Ferril what's the latest update on this issue?

**Comment by mackgorski** [comment]
What @ethanmdavidson mentioned is exactly what customer originally requested apart of marking these notifications as important as I explained several times already. I will make sure to update this issue description. This part was also originally discussed in customer ticket in https://github.com/grafana/support-escalations/issues/11449 I am sure we have this logged as FR as well to implement a proper Round-Robin like solution here. Le me find the FR for the second part of this use case.
